### PR TITLE
Boat Tweaking

### PR DESCRIPTION
**Pull request by mienaikoe**
- Using Eye icon instead of lock icon to show visibility
- Preventing dnd on mobile
- Using "--" to show an empty seat
- Paddler modal on mobile has every attendee, not just the ones who aren't on the boat yet. (This is because dnd was previously the only way to swap people)

### CODE REVIEW

File: client/components/BoatPanel/Seat.js [hunk 1]
```diff
@@ -94,8 +95,8 @@ class Seat extends Component {
       seatClass.push(styles.assigned);
     } else {
       children = <div className={styles.paddler}>
-        <span className={styles.desktop}>Drag to Assign</span>
-        <span className={styles.mobile}>Tap to Assign</span>
+        <span className={styles.desktop}>--</span>
+        <span className={styles.mobile}>--</span>
```
Comment (k3ntako) [inline client/components/BoatPanel/Seat.js]: issue with this is that now it's not clear how we are supposed to assign people on a user's first time. Should we have a tutorial?

File: client/components/BoatPanel/Seat.js [hunk 2]
```diff
@@ -126,7 +127,7 @@ class Seat extends Component {
       seatClass.push(styles.assigned);
     } else {
       children = <div className={styles.paddler}>
-        Not Assigned
+        --
```
Comment (k3ntako) [inline client/components/BoatPanel/Seat.js]: same as above.

File: client/components/BoatPanel/Seat.js (hunk 1 above)
Comment (mienaikoe) [reply client/components/BoatPanel/Seat.js]: yea. I'll add a tutorial.

**Comment by mienaikoe** [comment]
- Move Tabs into Card
- Dropdowns get cut off

File: client/components/Subheading/index.css [hunk 3]
```diff
@@ -9,4 +9,30 @@
...
+.collapse{
+  display: block;
+}
+
+.root.tabs h3{
+  text-align: center;
+  cursor: pointer;
+  /* font-size: 1.5rem;
+  line-height: 3rem;
+  margin: 1rem 0; */
```
Comment (k3ntako) [inline client/components/Subheading/index.css]: comment

File: client/pages/Boat/index.js [hunk 4]
```diff
@@ -116,6 +116,7 @@ class BoatPage extends Component {
 
       return <Tab key={boat.name} label={boat.name} icon={lock}>
         <BoatTab
+          team={me.team}
```
Comment (k3ntako) [inline client/pages/Boat/index.js]: Seems like `BoatTab` doesn't need `team`.

File: client/pages/Boat/BoatComparison.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+      events: [eventToSource(props.event)],
+      event: props.event,
+      boats: props.event.boats.map(boatToSource)
+    };
+    this.onEvent = this.onEvent.bind(this);
+    this.onBoat = this.onBoat.bind(this);
+  }
+
+  componentDidMount(){
+    Event.listForMyTeam("past").then( events => {
```
Comment (k3ntako) [inline client/pages/Boat/BoatComparison.js]: what if we only got the Events that have boats? Would that be confusing?

File: client/pages/Boat/index.js (hunk 4 above)
Comment (mienaikoe) [reply client/pages/Boat/index.js]: It was needed as it passed down team to the dock, which passed it to paddlers. But I can pass it directly from dock to paddlers and bypass this.

File: client/pages/Boat/BoatComparison.js (hunk 5 above)
Comment (mienaikoe) [reply client/pages/Boat/BoatComparison.js]: While it's possible, I think it's a bit of a corner case. Pretty much every event coach has should have boats.

**Pull request merged**